File: naive/fib.jl
function fib(n)
    if n == 1
        return 1
    end

    if n == 2
        return 1
    end

    return fib(n-1) + fib(n-2)
end

n = parse(Int64, ARGS[1])
println("res = ", fib(n))

File: naive/fib.rs
use std::env;

fn fib(n: i64) -> i64 {
    if n == 1 {
        return 1;
    }
    if n == 2 {
        return 1;
    }

    return fib(n-1) + fib(n-2);
}

fn main() {

    let args: Vec<String> = env::args().collect();

    let num = &args[1];
    let n: i64 = num.parse::<i64>().unwrap();

    let res: i64 = fib(n);

    println!("res {}", res);

}
